automation/common.sh

```diff
@@ -3,6 +3,16 @@
 # Common functions for the scripts
 #
 
+
+set_guestfs_params() {
+    # see: https://bugzilla.redhat.com/show_bug.cgi?id=1404287
+    export LIBGUESTFS_APPEND="edd=off"
+    # make libguestfs use /dev/shm as tmpdir
+    export LIBGUESTFS_CACHEDIR="/dev/shm"
```

Comment:
1. Actually looking at it now it uses /tmp as default which is tmpfs usually, so I wouldn't expect to see great improvements here. I don't think we should enforce any of LIBGUESTFS_* - if the user sets 'LIBGUESTFS_*' it should always be used(see https://github.com/lago-project/lago/pull/383). Using /dev/shm is not that standard.
2. Trying again with LIBGUESTFS_TMPDIR.. lets see if it yields better results. Next thing is to disable LIBGUESTFS_DEBUG in the functional tests.